src/store/index.ts

```diff
@@ -484,6 +485,14 @@ const {
 
       commit.setNetworkMetadata(networkMetadata);
       commit.setColumnTypes(networkMetadata);
+
+      // Guess the best label variable and set it
+      const allVars: Set<string> = new Set();
+      network.nodes.map((node: Node) => Object.keys(node).forEach((key) => allVars.add(key)));
+
+      const bestLabelVar = [...allVars]
+        .find((colName) => !isInternalField(colName) && context.getters.columnTypes[colName] === 'label');
+      commit.setLabelVariable(bestLabelVar);
```

Comment:
I agree that this doesn't belong in here. It should go in a separate action, and that action should be invoked explicitly from its point of use. This is important in case, e.g., the user has already set a label field and then (for any reason) they need to re-fetch the network. You wouldn't want the user's choice to be overridden with this guess in that case.

Reply:
See 285650c